examples/apps/cli/main.c

```diff
@@ -97,3 +98,22 @@ int main(int argc, char *argv[])
 
     return 0;
 }
+
+/*
+ * Provide, if required an "otPlatLog()" function
+ */
+#if ((OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_NONE) ||  (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_APP_STREAM))
+void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
```

Comment:
This is just question: 

When you try to run certification tests you have to pass some of MeshCoP logs into CLI output. Previously i just generated proper OT libraries (with DEFAULT_LOG_OUTPUT=1 and CERT_LOGS=1) and use one of our SDK example which includes CLI library.

Now we have to ensure that all SDK examples have to implement this function outside of OpenThread libraries. It's fine, but seems like adding unnecessary maintanance and documentation cost.

Could we somehow provide the default otPlatLog function inside OT libraries in case CERT_LOGS are enabled? (e.g. in openthread-cli.a as it was before) instead of demanding application to implement it?